hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/marker/DirectWriteMarkers.java

```diff
@@ -182,6 +182,7 @@ public Option<Path> createWithEarlyConflictDetection(String partitionPath, Strin
   private Option<Path> create(Path markerPath, boolean checkIfExists) {
     HoodieTimer timer = HoodieTimer.start();
     Path dirPath = markerPath.getParent();
+    boolean overwrite = false;
```

Comment:
You should not change this method directly, did you notice that there is a param `checkIfExists`, and I'm wondering first why the append handle could have duplicate marker files first?

Reply:
i notice the param checkIfExists, and if task fail but stage not retry，would running task with the same attamptid，so duplicate marker files would create